lua/entities/ent_jack_paintcan/cl_init.lua

```diff
@@ -121,6 +121,7 @@ function ENT:OpenTheMenu( tab )
     ColorPanel:SetSize( 40, 50 )
 
     ColorPanel.Paint = function()
+        if not IsValid( self ) then return end

```

Comment:
If the ColorPanel manages to be invalid but still running its Paint function, that sounds like a structural problem. It shouldn't need to do a validity check here.

Reply:
self isnt the paint panel, its the entity

Reply:
Then it should destroy the DermaPanel and ideally do that when the entity is removed, not in a Paint function.